pom.xml

```diff
@@ -26,10 +26,7 @@
             <groupId>org.springframework.boot</groupId>
```

Comment:
Row 18 is part of consul, not needed yet

Reply:
I'm talking about this row: 
`<spring-cloud.version>2021.0.2</spring-cloud.version>`
We don't need it right now.